src/components/steps/save/SaveStep.tsx

```diff
@@ -57,18 +58,21 @@ const SaveStep: React.FC<StepProps> = ({ project, onCancel, action }) => {
     );
 
     const checkExistingDataAndSave = React.useCallback(async () => {
```

Comment:
Check the linting warnings (shown on `yarn start` output or the JS console or explit `yarn lint`):

```Line 79:8:  React Hook React.useCallback has a missing dependency: 'loading'. Either include it or remove the dependency array```